spec/features/visitor_can_remove_items_from_cart_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature "visitor can remove items from cart" do
+  scenario "cart shows no items in cart after removal" do
+
+    db_repo = FactoryJordan.new
+    db_repo.create_travesty("Environmental Disasters")
+
+    item_1 = db_repo.items[0]
+
+    visit '/items'
+
+    first(:button, 'Add to cart').click
+
+    expect(page).to have_content("My Cart: 1")
+    expect(page).to have_content("You have 1 item in your cart.")
+
+    visit '/cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.